lib/pageflow/configuration.rb

```diff
@@ -476,5 +488,27 @@ class FeatureLevelConfiguration < Struct.new(:config)
       delegate :themes, to: :config
       delegate :file_importers, to: :config
     end
+
+    module EntryTypeConfiguration
+      def assign_config(value)
+        @config = value
+      end
```

Comment:
Renaming this to `initialize` and having configuration classes call `super(config)` would be more idiomatic, I think.